packages/flutter_tools/lib/src/native_assets.dart

```diff
@@ -415,3 +437,234 @@ Uri buildUriMultiple(Uri projectUri) {
   final String buildDir = build_info.getBuildDirectory();
   return projectUri.resolve('$buildDir/native_assets/multiple/');
 }
+
+Future<Uri?> dryRunNativeAssetsLinux({
```

Comment:
Optional: It's weird to have linux/native_assets.dart but for this not to be there; consider making the `*SingleArchitecture` helpers public after all, and then moving the wrappers to the platform-specific native_assets.dart files for consistency.